stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -372,7 +431,7 @@ private void executeTokenTask(Executor executor, AsyncTask<Void, Void, ResponseW
         return tokenParams;
     }
 
-    private class ResponseWrapper {
+    public class ResponseWrapper {
```

Comment:
Oh! It doesn't. I originally had the synchronous method returning a ResponseWrapper before I remembered that the error handling doesn't need to be contained in the synchronous method. Good catch!